spec/parliament/ntriple/utils_spec.rb

```diff
@@ -184,4 +184,24 @@
       expect(sorted_incumbencies[1].end_date).to eq(DateTime.new(2015, 3, 30))
     end
   end
+
+  describe '#multi_direction_sort' do
+    it 'returns a response sorted by member_count (desc) and name (asc)' do
```

Comment:
Done - there are now tests for asc asc, desc desc, desc asc and three parameters.